go.mod

```diff
@@ -10,29 +10,31 @@ require (
 	github.com/gofrs/uuid v4.2.0+incompatible
 	github.com/golang/protobuf v1.5.3
 	github.com/jessevdk/go-flags v1.5.0
-	github.com/stretchr/testify v1.8.1
+	github.com/stretchr/testify v1.8.2
 	go.uber.org/zap v1.24.0
-	google.golang.org/grpc v1.40.0
-	google.golang.org/protobuf v1.28.1
+	google.golang.org/grpc v1.55.0
+	google.golang.org/protobuf v1.31.0
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
-	k8s.io/api v0.27.2
-	k8s.io/apimachinery v0.27.2
-	k8s.io/client-go v0.27.2
+	k8s.io/api v0.28.2
+	k8s.io/apimachinery v0.28.2
+	k8s.io/client-go v0.28.2
 	sigs.k8s.io/sig-storage-lib-external-provisioner/v6 v6.3.0
 )
 
+require golang.org/x/term v0.10.0 // indirect
```

Comment:
@Bhagyashreek8 I know this seperate `require` block is getting added automatically. But can you fix it manually rather than adding a new `require` block?